.gitignore

```diff
@@ -10,6 +10,8 @@
 # Ignore the default SQLite database.
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/database.yml
```

Comment:
The database.yml file usually doesn't need to be changed. Although I can appreciate cloud9's set up may be different. Can you remove this from the .gitignore in this PR please Karen?